lib/facter/udevadm.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Fact: udevadm
+#
+# Purpose:
+#   Provide information about the udevadm utilty.
+#
+# Resolution:
+#   Check if the kernel is linux.
+#
+# Caveats:
+#
+Facter.add(:udevadm) do
+  confine(kernel: 'Linux')
+
+  setcode do
+    path = Facter::Core::Execution.which('udevadm')
```

Comment:
I prefer to confine on the file, but thats personal preference.

Reply:
Uh, is the path `/usr/bin/udevadm` guaranteed to exist?